apps/files/app.js

```diff
@@ -319,8 +319,8 @@ app.use(cloudcmd({
         ssh_hosts:              sshAppUrls(sshHosts(process.env.OOD_SSH_HOSTS), oodShellUrl()),
         fileexplorer_version:   app_version,
         // function that accepts a path and returns true or false
-        // FIXME: whitelist would be better as a function that has some properties!
-        whitelist: whitelist.enabled() ? whitelist : null
+        // FIXME: allowlist would be better as a function that has some properties!
+        allowlist: allowlist.enabled() ? allowlist : null
```

Comment:
You need to also change "whitelist" where it is used in the rest of the app.

Reply:
Should the documentation also be changed? (ex: changelog.md or //comments)